s/superagent/superagent_v3.8.3_rhel_8.5.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package           : superagent
+# Version           : v3.8.3
+# Source repo       : https://github.com/visionmedia/superagent
+# Tested on         : RHEL 8.5,UBI 8.5
+# Language          : Node
+# Travis-Check      : True
+# Script License    : Apache License, Version 2 or later
+# Maintainer        : Bhagat Singh <Bhagat.singh1@ibm.com>
+#
+# Disclaimer        : This script has been tested in root mode on given
+# ==========          platform using the mentioned version of the package.
+#                     It may not work as expected with newer versions of the
+#                     package and/or distribution. In such case, please
+#                     contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#Exit immediately if a command exits with a non-zero status.
+set -e
+
+PACKAGE_NAME=superagent
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-v3.8.3}
+PACKAGE_URL=https://github.com/visionmedia/superagent
+
+yum install -y yum-utils nodejs nodejs-devel nodejs-packaging npm git jq make cmake
```

Comment:
As far as I know, `yum install nodejs` installs node 10. We need to validate the node packages on nodejs 12 (minimum version).
Please made according changes and validate the package at least on node 12.

Reply:
This package is working fine with node 10x